drivers/docker/driver_test.go

```diff
@@ -1610,6 +1624,10 @@ func TestDockerDriver_OOMKilled(t *testing.T) {
 	}
 	testutil.DockerCompatible(t)
 
+	if runtime.GOOS == "windows" {
+		t.Skip("Windows does not support OOM Killer")
```

Comment:
I'm not sure what the problem here is.  This test times out now without being killed.  It might be because Windows doesn't enforce memory limits, or that our structs has some Linux-ism (e.g. LinuxResources) and we don't set fields properly on Windows.

Reply:
I'm _guessing_ that this is because in Hyper-V VMs you might get swap or something rather than strict limits